runtime-manager/src/lib.rs

```diff
@@ -12,6 +12,8 @@
 #![crate_name = "runtime_manager_enclave"]
 #![cfg_attr(feature = "sgx", no_std)]
 #![feature(rustc_private)]
+//TODO REMOVE. Use this feature for temporary glue code in fs.
```

Comment:
Similar comment as above about this TODO.  Please make sure they start getting tracked in the issue tracker.

Reply:
This is legacy code. Fixed.

Reply:
It's still there?